json-rpc/src/views.rs

```diff
@@ -179,21 +180,22 @@ impl TransactionView {
 pub enum ScriptView {
     #[serde(rename = "peer_to_peer_transaction")]
     PeerToPeer {
+        script_hash: String,
```

Comment:
if it's going to be common field for all enum values, do you think we should move it one level up to UserTransaction?

Reply:
done